lib/kitchen/provisioner/dsc.rb

```diff
@@ -24,12 +24,9 @@ class Dsc < Base
 
       default_config :configuration_script_folder, "examples"
       default_config :configuration_script, "dsc_configuration.ps1"
-      default_config :configuration_name do |provisioner|
-        provisioner.instance.suite.name
-      end
+      default_config :configurations, []
```

Comment:
This is a breaking change, which I'd like to avoid.  If we can keep the name and default the suite name as the config to run, we should.  Otherwise, this will break everyone using that behavior.

Reply:
Done.